products/chromeos.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Chrome OS
+category: os
+tags: linux-distribution
+iconSlug: googlechrome
+permalink: /chromeos
+alternate_urls:
+-   /chrome-os
+versionCommand: cat /etc/issue
+releasePolicyLink: https://chromeos.dev/en/education/chromeos-lts
```

Comment:
I'm a bit confused by the "Education" label. Is what's documented on https://chromeos.dev/en/education/chromeos-lts valid for all Chrome OS versions? Is there a similar page that does not labelled "Education" ?